system-addon/content-src/components/LinkMenu/LinkMenu.jsx

```diff
@@ -24,7 +31,8 @@ export class _LinkMenu extends React.PureComponent {
             props.dispatch(ac.UserEvent({
               event: userEvent,
               source,
-              action_position: index
+              action_position: index,
+              value: siteInfo
```

Comment:
Since "value" is an optional field, let's simply leave it out if `Object.keys(siteInfo).length === 0`.

Reply:
Will that ever be true? All cards must have a "type" history, bookmark or trending and all topsites have an icon type attribute associated with them.

Reply:
Yep, as long as we can guarantee that `siteInfo` always contains at least one key, that's fine.